To: vim_dev@googlegroups.com
Subject: Patch 7.3.410
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.410
Problem:    Compiler error for // comment. (Joachim Schmitz)
Solution:   Turn into /* comment */.
Files:	    src/message.c


*** ../vim-7.3.409/src/message.c	2012-01-20 20:44:38.000000000 +0100
--- src/message.c	2012-01-23 20:46:36.000000000 +0100
***************
*** 333,339 ****
      }
      else
      {
! 	buf[e - 1] = NUL;  // make sure it is truncated
      }
  }
  
--- 333,339 ----
      }
      else
      {
! 	buf[e - 1] = NUL;  /* make sure it is truncated */
      }
  }
  
*** ../vim-7.3.409/src/version.c	2012-01-20 21:13:55.000000000 +0100
--- src/version.c	2012-01-23 20:47:30.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     410,
  /**/


-- 
Are leaders born or made?  And if they're made, can we return them under
warranty?
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
